File: Exercicio Diagrama de Classes/Classes_Python/pessoa_juridica.py
from pessoa import Pessoa

class PessoaJuridica(Pessoa):
    def __init__(self, identificador: str, endereco: str, nome: str):
        super().__init__(identificador, endereco)
        self.nome = nome

    def exibir_dados(self):
        super().exibir_dados()
        print(f"Nome (Pessoa Jurídica): {self.nome}")

File: Classes_JS/PessoaJuridica.js
import { Pessoa } from "./Pessoa.js";

export class PessoaJuridica extends Pessoa {
  constructor(identificador, endereco, nome) {
    super(identificador, endereco);
    this.nome = nome;
  }

  exibirDados() {
    return `${super.exibirDados()}\nNome (Pessoa Jurídica): ${this.nome}`;
  }
}